integration/tests/cook/test_basic.py

```diff
@@ -29,6 +29,12 @@ def test_basic_submit(self):
         job = util.wait_for_job(self.cook_url, job_uuid, 'completed')
         self.assertEqual('success', job['instances'][0]['status'])
 
+    def test_disable_mea_culpa(self):
```

Comment:
Added tests for 1 & 2. Three I think will be difficult to reproduce in an integration test, but there is already a unit test for it.